packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -88,3 +89,35 @@ sizes.forEach((size) => {
         await percySnapshot(page, `Modal - size = ${size}`);
     });
 });
+
+test.describe('PIE Modal `isDismissible`', () => {
+    test.describe('when truthy', () => {
+        test('should display a close button within the modal', async ({ mount, page }) => {
+            await mount(PieModal, {
+                props: {
+                    heading: 'This is a modal heading',
+                    headingLevel: 'h2',
+                    isOpen: true,
+                    isDismissible: true,
+                },
+            });
+
+            await percySnapshot(page, 'Modal with close button displayed - isDismissible: `truthy`');
```

Comment:
```suggestion
            await percySnapshot(page, 'Modal with close button displayed - isDismissible: `true`');
```

IMO we should only really use truthy/falsey when we want values like `5`/`"hello"`/`{}`/`[]` to be valid and `0`/`''`/`null`/`undefined` to be invalid. For a boolean let's stick to `true`/`false`.